<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>491</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Sans</family>
      <pointsize>15</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TRABALHO - MÉTODOS NUMÉRICOS I - TEMA 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLambda">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ADICIONAR LAMBDA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>a2 =</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>a3 =</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>90</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>n =</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic</family>
      <pointsize>10</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Lambda =</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFuncao">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Stylus BT</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;f(d) = a&lt;/span&gt;&lt;span style=&quot; font-size:14pt; vertical-align:sub;&quot;&gt;3&lt;/span&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt; *d&lt;/span&gt;&lt;span style=&quot; font-size:14pt; vertical-align:super;&quot;&gt;3&lt;/span&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt; - 9a&lt;/span&gt;&lt;span style=&quot; font-size:14pt; vertical-align:sub;&quot;&gt;2&lt;/span&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt; d + 3&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableResultados">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>621</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnCalcular">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>320</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>140</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MaxIter =</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Epsilon =</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLimpar">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>320</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpar Tela</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputMaxIter">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>130</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputEps">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputA2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputA3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="vetorLambda">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>256</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputN">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputLambda">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLambdaRem">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>REMOVER LAMBDA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Valores de Lambda:</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>642</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnLimpar</sender>
   <signal>clicked()</signal>
   <receiver>inputMaxIter</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLimpar</sender>
   <signal>clicked()</signal>
   <receiver>inputA3</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLimpar</sender>
   <signal>clicked()</signal>
   <receiver>inputEps</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>481</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLimpar</sender>
   <signal>clicked()</signal>
   <receiver>inputA2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLimpar</sender>
   <signal>clicked()</signal>
   <receiver>tableResultados</receiver>
   <slot>clearContents()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>473</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>469</x>
     <y>410</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLimpar</sender>
   <signal>clicked()</signal>
   <receiver>tableResultados</receiver>
   <slot>clearSelection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>492</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLimpar</sender>
   <signal>clicked()</signal>
   <receiver>inputN</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
